app/controllers/account/companies_controller.rb

```diff
@@ -7,6 +7,10 @@ def index
 
   def show
     @company = current_company
+    twitter_link = @company.twitter
+    unless twitter_link.blank?
+      @tweets = TwitterProcessor.new(twitter_link: twitter_link, number_of_tweets: 6).process
```

Comment:
TwitterProcessor.new(company: , number_of_tweets

validate if name exists in processor file

if not valid @tweets return [] empty array